.github/workflows/check.yaml

```diff
@@ -0,0 +1,83 @@
+name: Check
+
+on:
+  pull_request:
+    branches: [ main ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  base:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout change
+        uses: actions/checkout@v2
+        with:
+          path: permian
+
+      - name: Build main container
+        run: |
+          pushd $GITHUB_WORKSPACE/permian
+          ./build_container
+          popd
+
+      - name: Run base tests
+        run: |
+          pushd $GITHUB_WORKSPACE/permian
+          ./in_container make test
+          popd
+
+      - name: Build documentation
+        run: |
+          pushd $GITHUB_WORKSPACE/permian
+          ./in_container make doc
+          popd
+
+      - name: Cleanup
+        if: always()
+        run: |
+          rm -rf $GITHUB_WORKSPACE/permian-redhat
+          podman rmi -f permian-redhat || true
```

Comment:
The permian-redhat seem to belong to the "base ("vanilla") cleanup, I guess this is just copy-paste remnant.

Reply:
Fixed